Identity Beta release June 2020 changelog updates (#11996)
Azure/azure-sdk-for-java: sdk/identity/azure-identity/CHANGELOG.md

## 1.1.0-beta.5 (Unreleased)

### New Features

> - Added option to write to shared token cache from `ClientSecretCredential`, `ClientCertificateCredential.

Reviewer: ```suggestion
- Added option to write to shared token cache from `ClientSecretCredential`, `ClientCertificateCredential`.
```

> - Added new Developer credentials `IntelliJCredential`, `VsCodeCredential` and `AzureCliCredential`.

Reviewer: ```suggestion
- Added new developer credentials `IntelliJCredential`, `VsCodeCredential` and `AzureCliCredential`.
```

> - First preview of new API for authenticating users with `DeviceCodeCredential` and `InteractiveBrowserCredential`

Reviewer: Probably don't need to call out that it is the first preview, can simply state

```suggestion
- New APIs for authenticating users with `DeviceCodeCredential` and `InteractiveBrowserCredential`.
```

> - Added method `authenticate` which pro-actively interacts with the user to authenticate if necessary and returns a serializable `AuthenticationRecord`

Reviewer: Was this new method added to a top level interface or to specific classes?

Author: Added a tab to this text, it is supposed to be under the previous comment.

- Added following configurable options in classes `DeviceCodeCredentialBuilder` and `InteractiveBrowserCredentialBuilder`
    - `authenticationRecord` enables initializing a credential with an `AuthenticationRecord` returned from a prior call to `Authenticate`
    - `disableAutomaticAuthentication` disables automatic user interaction causing the credential to throw an `AuthenticationRequiredException` when interactive authentication is necessary.

### Breaking Changes

> - Removed support to disable specific credentials in `DefaultAzureCredential` authentication flow.

Reviewer 2: Removed support to exclude specific credentials?
